lib/screens/event/screen.dart

```diff
@@ -0,0 +1,155 @@
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<String>(
+      stream: dropdownMenuStreamController.stream,
+      builder: (BuildContext context, AsyncSnapshot<String> optionVal) {
+        switch (optionVal.connectionState) {
+          case ConnectionState.waiting:
+            return Center(
+              child: CircularProgressIndicator(),
+            );
+          default:
+            return GestureDetector(
+              onTap: () => FocusScope.of(context).requestFocus(FocusNode()),
+              child: RefreshIndicator(
+                key: _refreshIndicatorKey,
+                onRefresh: load,
+                child: Column(
+                  children: <Widget>[
+                    Padding(
+                      padding: const EdgeInsets.only(
+                        left: 17.0,
+                        bottom: 10.0,
+                        right: 10.0,
+                      ),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: <Widget>[
+                          Flexible(
+                            flex: 9,
+                            child: SortDropdownMenu(
+                              preferences: 'eventsSortOption',
+                              streamController: dropdownMenuStreamController,
+                              value: optionVal.data,
+                              items: <String>[
+                                'Furthest',
+                                'Upcoming',
+                                'Most attendees',
+                                'Least attendees',
+                              ].map<DropdownMenuItem<String>>((String value) {
+                                Map<String, String> itemsMap = {
+                                  'Furthest': AppLocalizations.of(context)
+                                      .translate("furthest"),
+                                  'Upcoming': AppLocalizations.of(context)
+                                      .translate("upcoming"),
+                                  'Most attendees': AppLocalizations.of(context)
+                                      .translate("mostAttendees"),
+                                  'Least attendees':
+                                      AppLocalizations.of(context)
+                                          .translate("leastAttendees"),
+                                };
+                                return DropdownMenuItem<String>(
+                                  value: value,
+                                  child: Text(
+                                    itemsMap[value],
+                                  ),
+                                );
+                              }).toList(),
+                            ),
+                          ),
+                          Flexible(
+                            flex: 1,
```

Comment:
as a consequence to the russian forum sort dropdownmenu overflow error fix, the flex made the search button too small (on ray's phone). will this have similar implications on other phones ? ray's phone is android 5 so things might be different on other phones. regardless, test. the button seemed fine on flutter emulator, which is android 9/pie